AstSemantics.md

```diff
@@ -198,12 +200,11 @@ is why it isn't the specified default.)
 
 ### Out of bounds
 
-The ideal semantics is for out-of-bounds accesses to trap. A module may
-optionally define that "out of bounds" includes low-memory accesses.
+The ideal semantics is for out-of-bounds accesses to trap, but the performance
+implications and ability to polyfill to JS are not yet fully clear.
```

Comment:
On Thu, Jun 18, 2015 at 3:58 PM, Dan Gohman <notifications@github.com>
wrote:

> In AstSemantics.md
> <https://github.com/WebAssembly/design/pull/200#discussion_r32787666>:
>
> > @@ -198,12 +200,11 @@ is why it isn't the specified default.)
> >
> >  ### Out of bounds
> >
> > -The ideal semantics is for out-of-bounds accesses to trap. A module may
> > -optionally define that "out of bounds" includes low-memory accesses.
> > +The ideal semantics is for out-of-bounds accesses to trap, but the performance
> > +implications and ability to polyfill to JS are not yet fully clear.
>
> Polyfill considerations were specifically moved out to a separate document
> <http://Polyfill.md>.
>

That's fine. It just read funny previously. "The ideal semantics is for
XXX." And then a bulleted list of other potential things, leaving the
reader wondering why. Thus, a one sentence justification.

> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebAssembly/design/pull/200/files#r32787666>.
>